docs/topics/modules/interpreting_stack_report.adoc

```diff
@@ -0,0 +1,47 @@
+[#interpreting_stack_report]
+= Interpreting your Stack Report
+
+The Stack Report comprises three components:
+
+== Stack report: An overall stack analysis
+The Stack report scans the total number of components in your stack, and reports those analyzed by {osio} and those unknown to it.
+
+It displays:
+
+Security Alerts:: {osio} examines the Common vulnerabilities and exposures (CVEs) for the components of the stack and highlights vulnerabilities, if any.
+Usage:: It analyzes your overall stack composition, compares it to other similar stacks and displays the number of components in your stack that are not commonly used in other similar stacks or that rarely go together.
```

Comment:
"it analyzes"? What's "it"? OSIO, or Stack Report?

Reply:
This problem with "it" vs "OSIO" vs something else is in this entire list. It should be consistent.